ophyd/areadetector/plugins.py

```diff
@@ -419,7 +411,14 @@ class OverlayPlugin(PluginBase):
                              ('y', 'MaxSizeY_RBV'))),
                    doc='The maximum size in XY')
 
-    overlay_1 = C(Overlay, 'Overlay1:')
```

Comment:
Does this call for a DynamicDeviceCompoent? I don't super care, though. Everything else looks good. Self-merge at will.